### DISCUSSION TITLE
Thank you very much, have one enquiry

### DISCUSSION THREAD

**Discussion by nikhil-swamix** [ActivityWatch/activitywatch#1062] [General]
very good project, keep adding more features.
- how to get json data for custom dashboard implementation

**Comment by BelKed** [comment]
If you are familiar with Python, you can use the [`aw-client`](https://github.com/ActivityWatch/aw-client) library to retrieve the data from ActivityWatch. After processing the data in the Python script, you can use another library to communicate with your custom dashboard.

If you want to use another programming language, you can connect to ActivityWatch via the [API](https://docs.activitywatch.net/en/latest/api/rest.html) and the process would be similar.

**Comment by nikhil-swamix** [comment]
nice will tryout, i will consult if any difficulty faced. thank.
